transfers/pre-transfer/fix_unlv_checksums.py

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env python
+
+from __future__ import print_function
+
+import sys
+import os
+
+
+HASHFUNC_EXT = 'md5'
+
+
+def find_objects(path, ignore_dirs=None):
```

Comment:
Why not use os.walk and check if it matches/startswith the ignore_dirs (ignore_paths?) before yielding the value?